src/main/java/org/apache/maven/plugins/dependency/AbstractDependencyMojo.java

```diff
@@ -325,7 +325,7 @@ protected void unpack( Artifact artifact, String type, File location, String inc
         catch ( ArchiverException e )
         {
             throw new MojoExecutionException( "Error unpacking file: " + file + " to: " + location
-                + System.lineSeparator() + e.toString(), e );
+                + System.lineSeparator() + e, e );
```

Comment:
I consider this pointless to incude `e.toString()` in the exception message. One already passed `e`. That is more than enough.

Reply:
HI @michael-o
Agree. Removed.
TY